### DISCUSSION TITLE
[movement]make Line moving with Point

### DISCUSSION THREAD

**Discussion by suucss** [3b1b/manim#1560] [Q&A] [answered]
落霞与孤鹜齐飞，垂线与白点共动……
![QQ截图20210620092808](https://user-images.githubusercontent.com/84609589/122659380-3c39d280-d1aa-11eb-9865-ca22c5d43a8b.png)
The white point is an active point moving along on the Inverse proportion function: 1/x, what I want to solve is making the vertical yellow line moving together with the dot. The following picture displayed my code. How to add some codes to achieve my purpose?Thanks!
白点在反比例函数上运动，我想让黄线也伴随其运动，下面是代码，我应该添加什么代码来达到目的？谢谢！：）
![QQ截图20210620094649](https://user-images.githubusercontent.com/84609589/122659599-596fa080-d1ac-11eb-91dd-18b2d6810aab.png)

**Comment by TonyCrane** [comment, marked as answer]
Please paste the code instead of the screenshot.
> And please avoid using Chinese here.

**Reply by suucss** [reply]
> f1 = self.get_graph(x1, color=YELLOW, x_min=0.25, x_max=5)
        f1name = MathTex("y=\\frac{1}{x}").shift(self.coords_to_point(3, 3))
        dot1 = Dot(point=self.coords_to_point(0.25, 4))
        line1 = self.get_vertical_line_to_graph(0.25, f1, Line)
        self.play(Create(f1), FadeInFrom(f1name, DOWN))
        self.play(FadeInFromLarge(dot1)), self.play(FocusOn(self.coords_to_point(0.25, 4)))
        self.play(Create(line1))
        self.play(MoveAlongPath(dot1, f1), run_time=3, rate_functions=linear)

Requirements read, Okay~

**Reply by TonyCrane** [reply]
Maybe `line1.add_updater(lambda m: m.become(self.get_vertical_line_to_graph(dot1.get_x(), f1, Line)))`.
But I noticed that you are using the community version, so you'd better ask the question in their discord channel or open an issue in [ManimCommunity/manim](https://github.com/ManimCommunity/manim) instead of here.

**Reply by suucss** [reply]
Thanks for your enthusiastic help! I get it!
